bucket/freepascal.json

```diff
@@ -0,0 +1,31 @@
+{
+    "homepage": "https://www.freepascal.org",
+    "version": "3.2.2",
+    "description": "Professional Pascal compiler.",
+    "license": {
+        "identifier": "GPL-2.0-or-later | FPC-modified-LGPL-2.0-or-later",
+        "url": "http://wiki.lazarus.freepascal.org/licensing"
+    },
+    "url": "https://downloads.sourceforge.net/project/freepascal/Win32/3.2.2/fpc-3.2.2.i386-win32.exe",
+    "hash": "sha1:c3fc24290da7349e4fba763eaadd2a5bec07bfa3",
+    "innosetup": true,
+    "bin": [
+        "bin\\i386-win32\\fp.exe",
+        "bin\\i386-win32\\fpc.exe"
+    ],
+    "shortcuts": [
+        [
+            "bin\\i386-win32\\fp.exe",
+            "Free Pascal IDE",
```

Comment:
Does it belong to Extras instead if it also has an IDE?

Reply:
I think IDE is more like an "addition" to the main program.
Similar to [latex](https://github.com/ScoopInstaller/Main/blob/master/bucket/latex.json).